README.md

```diff
@@ -268,7 +268,7 @@ vcpkg install --overlay-ports=<path to the vcpkg-custom-ports above>
 If you are building using CMAKE, you can instruct CMAKE to apply the overlay ports using the following command line switches:
 
 ```sh
-vcpkg -DVCPKG_MANIFEST_MODE=ON -DVCPKG_OVERLAY_PORTS=<path to the vcpkg-custom-ports above> -DVCPKG_MANIFEST_DIR=<path to the directory containing the vcpkg.json file>
+cmake -DVCPKG_MANIFEST_MODE=ON -DVCPKG_OVERLAY_PORTS=<path to the vcpkg-custom-ports above> -DVCPKG_MANIFEST_DIR=<path to the directory containing the vcpkg.json file>
```

Comment:
I was initially having trouble finding docs for `VCPKG_MANIFEST_MODE`
I didn't see `VCPKG_MANIFEST_MODE` in the vcpkg Environment variable docs:
https://learn.microsoft.com/en-us/vcpkg/users/config-environment

Here's the cmake integration page lists these variables:
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration

Reply:
@LarryOsterman or @ByteYue please feel free to resolve this conversation, before merging.